change/@fluentui-react-make-styles-490e8040-40b2-45ae-bd61-1bd15ea5e794.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

```suggestion
  "type": "none",
```

This PR probably should not bump package version